app/src/main/java/com/teamounce/ounce/main/BottomSheetAdapter.kt

```diff
@@ -1,52 +1,101 @@
 package com.teamounce.ounce.main
 
+import android.app.Activity
 import android.content.Context
+import android.content.ContextWrapper
 import android.view.LayoutInflater
-import android.view.View
 import android.view.ViewGroup
-import android.widget.RadioButton
-import android.widget.TextView
+import android.widget.CompoundButton
 import androidx.recyclerview.widget.RecyclerView
-import com.teamounce.ounce.R
 import com.teamounce.ounce.databinding.ItemBottomMainBinding
-import com.teamounce.ounce.settings.ui.SettingCareCatData
-import kotlinx.android.synthetic.main.item_bottom_main.view.*
+import com.teamounce.ounce.util.SharedPreferences
 
-class BottomSheetAdapter : RecyclerView.Adapter<BottomSheetAdapter.BottomSheetViewHolder>() {
-
-    var settingCareCatData = mutableListOf<SettingCareCatData>()
-    var lastSelectionPosition = -1
-
-    override fun getItemCount(): Int {
-        return settingCareCatData.size
-    }
+class BottomSheetAdapter(private var context: Context) :
+    RecyclerView.Adapter<BottomSheetAdapter.BottomSheetViewHolder>() {
+    val mContext = getActivity(context)
+    var bottomSheetProfileData = mutableListOf<BottomSheetProfileData>()
```

Comment:
```suggestion
    private var bottomSheetProfileData = mutableListOf<BottomSheetProfileData>()
```

변동성이 있는 객체를 var로 선언해주는건 코틀린의 철학하고는 배치되게 작성하고 있는 것 같아. 그럴 수도 있다고 쳐도 private 선언도 안되어있으면 외부에서 이 객체를 쥐락펴락할 수도 있을텐데 만약에 내가 실수로 이 객체에 접근해서 값을 바꾼다고 하면 어떠한 결과를 초래할 수 있을까? 물론 내가 그럴 일은 드물테지만 이것도 엄연히 일어날 수 있는 가능성이 있어. 

코드를 작성할 때 가능성을 원천봉쇄할 수 있다면 원천봉쇄를 할 수 있게, 즉 실수를 허용 못하게 하는 코드가 제일 좋다고 생각해(이건 나의 의견이 아니라 실력이 뛰어나신 프로그래머분들의 의견을 차용한거야)